src/global/mixins/mixins_typo.scss

```
@@ -142,3 +142,10 @@
     color: var(--color-metal-default);
   }
 }
+
+@mixin clampLinesByTwo() {
```

I'd suggest to introduce a parameter for the count of lines
```suggestion
@mixin clampLinesBy($lines) {
```